src/NUnitFramework/framework/Api/NUnitTestAssemblyRunner.cs

```diff
@@ -159,6 +159,7 @@ public ITest Load(Assembly assembly, IDictionary settings)
         /// Count Test Cases using a filter
         /// </summary>
         /// <param name="filter">The filter to apply</param>
+        /// <exception cref="InvalidOperationException">The CountTestCases method was called but no test has been loaded</exception>
         /// <returns>The number of test cases found</returns>
         public int CountTestCases(ITestFilter filter)
```

Comment:
Called by NUnitLite and tests, internally by FrameworkController - same goes for all public methods of this class.